### PR TITLE
SALTO-5179: Change project's path

### PR DESCRIPTION
**Pull request by idoamit8**
In this PR I changed the projects path in jira. 

---

_Additional context for reviewer_
Paths view:
image

---
_Release Notes_: 
_Jira Adapter:_
Projects path are going to be changed to Projects -> {projects Type} ->...

---
_User Notifications_: 
_Jira Adapter:_
Projects path are going to be changed to Projects -> {projects Type} ->...

### CODE REVIEW

File: packages/jira-adapter/test/filters/project_component.test.ts [hunk 1]
```diff
@@ -81,6 +96,15 @@ describe('projectComponentFilter', () => {
...
     })
+    it('should add parent path', async () => {
+      await filter.onFetch([type, instance])
+      expect(instance.path).toEqual([JIRA, elementUtils.RECORDS_PATH, 'Project', 'Software', 'proj1', 'components', 'instance'])
+    })
+    it('should not change path if there is no parent', async () => {
+      delete instance.annotations[CORE_ANNOTATIONS.PARENT]
+      await filter.onFetch([type, instance])
+      expect(instance.path).toEqual([JIRA, elementUtils.RECORDS_PATH, 'Project', 'Software', 'proj1', 'components', 'instance'])
+    })
```
[the remark this answers is not in the document]
Comment (idoamit8) [reply packages/jira-adapter/test/filters/project_component.test.ts:107]: You are absolutely right! Changed it :)

File: packages/jira-adapter/src/filters/project.ts [hunk 2]
```diff
@@ -46,8 +46,11 @@ const changeProjectPath = (
   instance: InstanceElement,
   projectTypesKeysToFormatedKeys: Record<string, string>
 ): void => {
+  if (instance.path === undefined) {
+    return
```
Comment (omrilit) [inline packages/jira-adapter/src/filters/project.ts:50]: please add a warn log here

File: packages/jira-adapter/src/filters/project.ts [hunk 3]
```diff
@@ -42,6 +42,17 @@ const CUSTOMER_PERMISSIONS = 'customerPermissions'
...
 
+const changeProjectPath = (
+  instance: InstanceElement,
+  projectTypesKeysToFormatedKeys: Record<string, string>
+): void => {
+  if (instance.path === undefined) {
+    return
+  }
+  const subPath = projectTypesKeysToFormatedKeys[instance.value.projectTypeKey] ?? 'Other'
+  instance.path = [...instance.path?.slice(0, -2), subPath, ...instance.path?.slice(-2,)]
```
Comment (omrilit) [inline packages/jira-adapter/src/filters/project.ts:53]:
```suggestion
  instance.path = [...instance.path.slice(0, -2), subPath, ...instance.path.slice(-2,)]
```

**Review by omrilit — APPROVED** [review]
Looks great!
Approving to save cycle, please apply the same comments also to `project_component.ts`

**Pull request merged**